engine/api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/CustomAction.scala

```diff
@@ -0,0 +1,17 @@
+package pl.touk.nussknacker.engine.api.deployment
+
+import pl.touk.nussknacker.engine.api.process.ProcessName
+
+import java.net.URI
+
+case class CustomAction(name: String,
```

Comment:
let's add some comments that this is experimental/not final version?

Reply:
:+1:  done